app/components/primer/clipboard_copy.rb

```diff
@@ -34,10 +34,16 @@ def initialize(value: nil, **system_arguments)
       @system_arguments[:value] = value if value.present?
     end
 
+    # :nodoc:
+    def render_in(*args, &block)
+      validate_aria_label unless block_given?
```

Comment:
Hmm, I'm not sure checking the presence of a block is the right approach here. The component could ostensibly be passed an empty block, or someday have slots that must be defined in the block. What would you think about something like:

```ruby
def before_render
  validate_aria_label if content.blank?
end
```

Reply:
They can always give some content that is *not* suitable for screen readers, but explicitly checking for blank sounds good. `before_render` also sounds like a better place for this check than hooking into `render_in` as well.